ci-operator/config/openshift/release/openshift-release-master__ocp-4.5-ci.yaml

```diff
@@ -19,6 +19,21 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-upgrade-aws
+- as: e2e-openstack-osuosl-ipi
+  cron: 0 8,20 * * *
+  steps:
+    cluster_profile: openstack-osuosl
+    test:
+    - as: e2e
+      commands: |-
+        #!/bin/bash
+        e2e-tests # as built by go test -c
```

Comment:
Is it needed to put this command in a bash script?

Reply:
Good question @droslean .  Maybe @stevekuznetsov knows definitively.  I took this example from elsewhere.

Reply:
No, there's no need. Better left without